config/locales/pets.en.yml

```diff
@@ -0,0 +1,33 @@
+en:
+  errors:
+    messages:
```

Comment:
Internationalization for the active record validations gem is scoped like this. Now, we only use this gem right now on the Pet CRUD, so they are included in this file. However, I am inclined to move this to the main `en.yml` file as we are likely to want to enable uploads on other models in futures.